packages/eslint-plugin-turbo/lib/configs/recommended.ts

```diff
@@ -7,7 +7,7 @@ const envVars = getEnvVarDependencies({
 });
 const settings = {
   turbo: {
-    envVars: envVars ? [...envVars] : [],
+    envVars: envVars ? [...envVars].sort() : [],
```

Comment:
I realized that we should _probably_ make these stable. I don't know if ESLint's hash does it for us, but now I don't have to worry about it.